### ISSUE TITLE
Subscription payment collection.

### ISSUE THREAD

**Issue by michaelplzno** [mesosoi/silverwaregames-io-issue-tracker#364]
I'm not sure what system would be good to collect money, paypal, Bank of America payment portal, etc. but someday we want to take payments directly from our website rather than through facebook.

**Comment by Miragebell** [comment]
Right. Well, you have to choose it yourself here. I'll integrate anything you want.

> I'm not sure what system would be good

Just google "card acquiring". There should be a lot of systems to do that. For example, ECOMMPAY: https://ecommpay.com/shop/selection/

**Comment by michaelplzno** [comment]
It's not too pressing right now to integrate a pay portal because there are a lot of issues we need to take care of first.

**Comment by michaelplzno** [comment]
The ideal solution would be to let people process a payment as an IAP on their smartphones for a native app. I think pretty much everyone can figure out how to do that because the phone payment portals are clear and easy to use. I do have a pro paypal account that should be able to get the hooks needed to checkout via paypal.
